.gitmodules

```diff
@@ -1,28 +1,22 @@
-[submodule "docs/examples/quickstart/oso-java-quickstart"]
-	path = docs/examples/quickstart/oso-java-quickstart
+[submodule "docs/examples/roles/sqlalchemy/basic"]
+	path = docs/examples/roles/sqlalchemy/basic
+	url = https://github.com/osohq/oso-sqlalchemy-roles-guide.git
+	branch = basic_roles_complete
+[submodule "docs/examples/quickstart/go"]
+	path = docs/examples/quickstart/go
+	url = https://github.com/osohq/oso-go-quickstart.git
+[submodule "docs/examples/quickstart/java"]
+	path = docs/examples/quickstart/java
 	url = https://github.com/osohq/oso-java-quickstart.git
-        branch = main
```

Comment:
This doesn't work for me without specifying all the branches as main.

Reply:
ah right; I think that's due to a difference in Git versions

Reply:
fixed in b83a9c9c

Reply:
Thank you kindly